packages/core-api/src/routing/types.ts

```diff
@@ -16,26 +16,51 @@
 
 import { IconComponent } from '../icons';
 
-export const resolveRoute = Symbol('resolve-route');
-export const routeReference = Symbol('route-ref');
-
-export type ReferencedRoute = {
-  [routeReference]: unknown;
-};
-
-export type ConcreteRoute = ReferencedRoute & {
-  [resolveRoute](path: string): string;
-};
-
-export type RouteRef = {
+// @ts-ignore, we're just embedding the Params type for usage in other places
```

Comment:
Is this still needed?

Reply:
I got rid of the `.P` as I though it was best not to add that to the public API (yet), which makes this needed as TS will complain about the unused `Params` otherwise.